challenge.rb

```diff
@@ -1,26 +1,38 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  array = Array.new(input)
+
+  array.map! { |i| i.capitalize }
```

Comment:
The `map` function (without the !) returns the mutated array, and since Ruby will implicitly return the last line in your function, you can cut this method down by a couple of lines.